package.json

```diff
@@ -29,11 +29,14 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
+    "rxjs": "^5.5.0",
     "tap-spec": "^4.1.1",
     "tape": "^4.8.0"
   },
   "dependencies": {
-    "rxjs": "^5.5.0",
     "xhr2": "^0.1.4"
+  },
+  "peerDependencies": {
+    "rxjs": "^5.5.0"
```

Comment:
shouldn't `"rxjs": "^5.0"` be sufficient? If they stick to [semver](https://semver.org/) breaking changes should come with a change of the major version?! See https://docs.npmjs.com/files/package.json#peerdependencies

Reply:
I think that's a good remark. Maybe we can just do `"rxjs": "5.x"` like in the link you provided with `tea-latte` example. I will try that and update PR.